utils/mkhtml.py

```diff
@@ -418,11 +418,9 @@ def get_last_git_commit(src_dir, addon_path, is_addon):
             return get_git_commit_from_file(src_dir=src_dir)
 
 
-html_page_footer_pages_path = (
-    os.getenv("HTML_PAGE_FOOTER_PAGES_PATH")
+html_page_footer_pages_path = os.getenv("HTML_PAGE_FOOTER_PAGES_PATH")
```

Comment:
This is the same type of unrelated fix as before. ...and several issues are reported here (which I guess is a good thing).

Reply:
done